unity-client/Assets/ABConverter/AssetBundleMenuItems.cs

```diff
@@ -116,42 +134,9 @@ public static void DumpCenterPlaza()
         }
 
         [MenuItem("Decentraland/Asset Bundle Builder/Only Build Bundles")]
-        public static void OnlyBuildBundles() { BuildPipeline.BuildAssetBundles(ABConverter.Config.ASSET_BUNDLES_PATH_ROOT, BuildAssetBundleOptions.UncompressedAssetBundle | BuildAssetBundleOptions.ForceRebuildAssetBundle, BuildTarget.WebGL); }
-
-        public class WearableItemArray
-        {
-            public List<WearableItem> data;
-        }
-
-        public static MappingPair[] GetAvatarMappingList(string url)
+        public static void OnlyBuildBundles()
         {
-            List<MappingPair> mappingPairs = new List<MappingPair>();
-
-            UnityWebRequest w = UnityWebRequest.Get(url);
-            w.SendWebRequest();
-
-            while (!w.isDone) { }
-
-            if (!w.WebRequestSucceded())
-            {
-                Debug.LogWarning($"Request error! Parcels couldn't be fetched! -- {w.error}");
-                return null;
-            }
-
-            var avatarApiData = JsonUtility.FromJson<WearableItemArray>("{\"data\":" + w.downloadHandler.text + "}");
-
-            foreach (var avatar in avatarApiData.data)
-            {
-                foreach (var representation in avatar.representations)
-                {
-                    foreach (var datum in representation.contents)
-                    {
-                        mappingPairs.Add(datum);
-                    }
-                }
-            }
-
-            return mappingPairs.ToArray();
+            BuildPipeline.BuildAssetBundles(ABConverter.Config.ASSET_BUNDLES_PATH_ROOT, BuildAssetBundleOptions.UncompressedAssetBundle | BuildAssetBundleOptions.ForceRebuildAssetBundle, BuildTarget.WebGL);
```

Comment:
This change will provoke a conflict in my PR: https://github.com/decentraland/explorer/pull/2256 (related to change the use of `UnityWebRequest` by our new WebRequest system). Don't worry about that, I left this comment here only to say to Santi of the future that will have to take it on to account :D